src/toolbar_editor.cpp

```diff
@@ -28,10 +28,13 @@ Q_DECLARE_METATYPE(QMenu*)
 Q_DECLARE_METATYPE(QToolBar*)
 Q_DECLARE_METATYPE(QAction*)
 
+const QString Toolbar_Editor::customToolbarNamePrefix = QString("custom_toolbar_");
```

Comment:
I needed that information externally available to see which toolbars are custom toolbars.